tests/functional_tests/io-tests.bats

```diff
@@ -3,41 +3,55 @@
 # all the tests related to HackerRank compatible IO tests for DSA course
 # setup and teardown functions
 setup() {
-	echo "empty setup"
+	sed -i 's/unit_tests/io_tests/' ../../execution_nodes/extract_run.sh
+	mkdir $BATS_TMPDIR/io-tests-example
 }
 
 teardown() {
-	echo "empty teardown"
+    sed -i 's/io_tests/unit_tests/' ../../execution_nodes/extract_run.sh
+	rm -rf $BATS_TMPDIR/io-tests-example
```

Comment:
This line if parsed correctly by shellcheck gives the error:
Double quote to prevent globbing and word splitting. [SC2086]

an improved line:
rm -rf "$BATS_TMPDIR/io-tests-example"